.github/workflows/rust.yml

```diff
@@ -99,3 +99,44 @@ jobs:
       run: cargo build --verbose --features vendored
     - name: Run tests
       run: cargo test --verbose --features vendored
+
+  build-vendored-mingw64:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cargo/bin/
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          ~/.cargo/git/db/
+          target/
+        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
This cache key only from runner.os doesn't work well on multiple build environment in the same os like this PR.

Fix this later.

Reply:
To embed env.GITHUB_JOB for this purpose

Reply:
This didn't speed up CI build, will revert them...

Reply:
Reverted in 8d002036e1fbcc77e4e405cdc3e1591c6434086c